Update cookie macros about links styled as buttons (#2168)
alphagov/govuk-frontend: CHANGELOG.md

## New features

#### Allow users to use cookie banner buttons as links

You can now add links styled as buttons to cookie banners.

Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.

> - the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden

Author: ```suggestion

```

## New features

#### Allow users to use cookie banner buttons as links

You can now add links styled as buttons to cookie banners.

Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
- your cookie banner needs to work without JavaScript

> - the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden

Author: ```suggestion

```

## New features

#### Allow users to use cookie banner buttons as links

You can now add links styled as buttons to cookie banners.

> Use this feature only if you need to. For example, to display the **Hide** button because:

Author: ```suggestion
Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
```

> - your cookie banner needs to work without JavaScript

Author: Does the **Hide** button not display for non-JS users unless they style links as buttons?

> 

Author: ```suggestion
```

> - your cookie banner needs to work without JavaScript

Author: ```suggestion
```

- the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden

> This was added in [pull request #2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)

Author: 'Link' makes me think of navigation to another webpage. What, exactly, do we mean by 'link' in the context of the cookie banner?
